Dockerfile

```diff
@@ -8,15 +8,20 @@ ARG userid=10001
 COPY --from=ghcr.io/astral-sh/uv:0.6.14@sha256:3362a526af7eca2fcd8604e6a07e873fb6e4286d8837cb753503558ce1213664 /uv /uvx /bin/
 
 # Set environment variables
-ENV PYTHONUNBUFFERED=True
-ENV UV_COMPILE_BYTECODE=1
-ENV APP_HOME=/app
+ENV PYTHONUNBUFFERED=True \
+    PYTHONPATH=/app \
+    UV_COMPILE_BYTECODE=1 \
+    APP_HOME=/app
```

Comment:
This reduces the steps and thus the number of layers. It also adds a `PYTHONPATH` which allows things in the container to import Python modules from `/app` directory.